api/filter/network/http_connection_manager.proto

```diff
@@ -169,6 +169,10 @@ message HttpConnectionManager {
 
     // Whether to forward the SAN of the client cert. Defaults to false.
     google.protobuf.BoolValue san = 2;
+
+    // [#not-implemented-hide:]
+    // Whether to forward the entire client cert in PEM format. Defaults to false.
```

Comment:
Does this require base64 encoding in the header? Should we specify that in the docs?

Reply:
So right now it puts the entire PEM cert with \n and all between each line into the header.  I'm tweaking my other pull request to just put raw bytes base 64 encoded into the header rather than the PEM format.  Once I'm certain that works and gets around the newlines and such, I'll update this PR to change the documentation a bit to call that out explicitly.

Reply:
+1 I think that is safer and more portable.